### ISSUE TITLE
android delete counter is wrong

### ISSUE THREAD

**Issue by HolgerJeromin** [mapillary/mapillary_issues#1698]
take a sequence with 3 images.
review it
mark 2 images
click delete icon: "do you want to delete 2 images?" yes
mark the last one
click delete icon: "do you want to delete 3 images?" UPS. Cancel
unmark the last one
mark the last one
click delete icon: "do you want to delete 4 images?" UPS

leaving the sequence resets the counter

**Comment by AnkEric** [comment]
Finally, I succeeded in finding a bug that's reproducible en simple to resolve.
But, search for "Delete" issues and I found out it would be a duplicated issue.
But also, this simple issue is now open for 8 months and does not have a "Bug" status.
I agree, it’s not the most important issue. However, a fix would be appreciated! One line of code will do.

![screenshot_20161005-143507](https://cloud.githubusercontent.com/assets/20150550/19113660/fc562f8c-8b0a-11e6-9cf2-ae04bd5cc8de.png)
